akka-operator/values.yaml

```diff
@@ -2,11 +2,15 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+defaultAWSRepository: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lightbend/akka-cloud-platform
+defaultGCPRepository: gcr.io/lightbend-public/akka-cloud-platform
+# defaultGCPRepository: eu.gcr.io/bubbly-observer-178213/akka-operator
 image:
-  repository: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lightbend/akka-cloud-platform
+  repository: false
```

Comment:
do we agree on defaulting to AWS to give the best experience to users?
cc. @patriknw @RayRoestenburg @chbatey @seglo

Reply:
SGTM

@andreaTP Is this `defaultGCPRepository` the private one we talked about before?

Reply:
`defaultGCPRepository` basically doesn't exists, we can omit it entirely

Reply:
@andreaTP deleted it from `Values`

Reply:
A working equivalent to:
`repository: defaultAWSRepository` ?